resources/views/admin/feedbacks/create.blade.php

```diff
@@ -0,0 +1,116 @@
+@include('templates.header')
+@include('templates.admin-nav')
+
+<h2>Add Feedback</h2>
+
+@include('templates.flash-message')
+
+<div class="row">
+    <div class="col">
+        <label for="createdAt" class="form-label">Created At: </label>
+        <input
+            type="text"
+            class="form-control"
+            readonly
+            name="date"
+            value="{{ $worklog->created_at->format('Y-m-d, h:i A') }}"
```

Comment:
i have seen that  u have been using this date format in lot of places.
for this as well u can define a const DEFAULT_DATE_TIME_FORMAT and use in all places
in future if there is change is default date time fomat, u can simply upadte it in single place only